docs/administration/back_office/content_tab_switcher.md

```diff
@@ -1,4 +1,4 @@
-# Add tab switcher
+# Tab switcher in Content Type
```

Comment:
As discussed in private, this page is documenting tab switcher on content edit page.